html-report/src/components/SearchBar.js

```diff
@@ -1,21 +1,128 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import elasticlunr from 'elasticlunr';
 
+const SEARCH_FIELDS = ['package_name', 'class_name', 'name', 'id', 'status'];
+const SEARCH_REF = 'id';
+const EL_SEARCH = elasticlunr();
+const STATUSES = ['failed', 'ignored'];
```

Comment:
@artem-zinnatullin I know, but this was the quickest implementation after yesterday's discussion. To code it like you request I need a some more time :)